packages/ubuntu_desktop_installer/lib/l10n/app_en.arb

```diff
@@ -1,34 +1,59 @@
 {
   "@@locale": "en",
-
   "appTitle": "Ubuntu Desktop Installer",
-  "windowTitle": "Install Ubuntu",
-
+  "windowTitle": "Install {RELEASE}",
+  "@windowTitle": {
+    "type": "text",
+    "placeholders": {
+      "RELEASE": {}
+    }
+  },
   "cancelButtonText": "Cancel",
   "changeButtonText": "Change",
   "okButtonText": "OK",
   "noButtonText": "No",
   "restartButtonText": "Restart",
   "revertButtonText": "Revert",
   "yesButtonText": "Yes",
-
   "welcome": "Welcome",
-
   "tryOrInstallPageTitle": "Try or install",
   "repairInstallation": "Repair installation",
   "repairInstallationDescription": "Repairing will reinstall all installed software without touching documents or settings.",
-  "tryUbuntu": "Try Ubuntu",
-  "tryUbuntuDescription": "You can try Ubuntu without making any changes to your computer.",
-  "installUbuntu": "Install Ubuntu",
-  "installUbuntuDescription": "Install Ubuntu alongside (or instead of) your current operating system. This shouldn't take too long.",
+  "tryUbuntu": "Try {RELEASE}",
+  "@tryUbuntu": {
+    "type": "text",
+    "placeholders": {
+      "RELEASE": {}
+    }
+  },
+  "tryUbuntuDescription": "You can try {RELEASE} without making any changes to your computer.",
+  "@tryUbuntuDescription": {
+    "type": "text",
+    "placeholders": {
+      "RELEASE": {}
+    }
+  },
+  "installUbuntu": "Install {RELEASE}",
+  "@installUbuntu": {
+    "type": "text",
+    "placeholders": {
+      "RELEASE": {}
+    }
+  },
+  "installUbuntuDescription": "Install {RELEASE} alongside (or instead of) your current operating system. This shouldn't take too long.",
+  "@installUbuntuDescription": {
+    "type": "text",
+    "placeholders": {
+      "RELEASE": {}
+    }
+  },
   "releaseNotesLabel": "You may wish to read the <a href=\"{url}\">release notes</a>.",
   "@releaseNotesLabel": {
     "type": "text",
     "placeholders": {
       "url": {}
     }
   },
-
```

Comment:
the Flutter Intl extension for VS Code wants to remove these line-breaks, and Weblate doesn't retain them either, so...